assimilation_code/modules/io/io_filenames_mod.f90

```diff
@@ -1141,111 +1187,12 @@ function get_copy_name(file_handle, cnum)
 end function get_copy_name
 
 
-!#! !------------------------------------------------------------------
-!#! !> set netcdf file type
-!#!
-!#!
-!#! subroutine nc_set_netcdf_info(file_handle, ncFileInfo)
-!#!
-!#! type(file_info_type),   intent(inout) :: file_handle
-!#! type(netcdf_file_type), intent(in)    :: ncFileInfo
-!#!
-!#! file_handle%stage_metadata%ncFileID = ncFileInfo
-!#!
-!#! end subroutine nc_set_netcdf_info
-!#!
-!#!
-!#!
-!#!
-!#! !------------------------------------------------------------------
-!#! !> get diagnostic id
-!#!
-!#!
-!#! subroutine nc_set_diag_id(ncFileInfo, diagnostic_id)
-!#!
-!#! type(netcdf_file_type), intent(inout) :: ncFileInfo
-!#! integer,                intent(in)    :: diagnostic_id
-!#!
-!#! ncFileInfo%diag_id = diagnostic_id
-!#!
-!#! end subroutine nc_set_diag_id
-!#!
-!#!
-!#! !------------------------------------------------------------------
-!#! !> get diagnostic id
-!#!
-!#!
-!#! function nc_get_diag_id(ncFileInfo)
-!#!
-!#! type(netcdf_file_type), intent(in)    :: ncFileInfo
-!#! integer :: nc_get_diag_id
-!#!
-!#! nc_get_diag_id = ncFileInfo%diag_id
-!#!
-!#! end function nc_get_diag_id
-!#!
-!#!
-!#! !------------------------------------------------------------------
-!#! !> set netcdf file name
-!#!
-!#!
-!#! subroutine nc_set_fname(ncFileInfo, filename)
-!#!
-!#! type(netcdf_file_type), intent(inout) :: ncFileInfo
-!#! character(len=*) :: filename
-!#!
-!#! ncFileInfo%fname = filename
-!#!
-!#! end subroutine nc_set_fname
-!#!
-!#!
-!#! !------------------------------------------------------------------
-!#! !> get netcdf file name
-!#!
-!#!
-!#! function nc_get_fname(ncFileInfo)
-!#!
-!#! type(netcdf_file_type), intent(in)    :: ncFileInfo
-!#! character(len=80) :: nc_get_fname
-!#!
-!#! nc_get_fname = ncFileInfo%fname
-!#!
-!#! end function nc_get_fname
-!#!
-!#!
-!#! !------------------------------------------------------------------
-!#! !> get netcdf file id
-!#!
-!#!
-!#! subroutine nc_set_ncid(ncFileInfo, my_ncid)
-!#!
-!#! type(netcdf_file_type), intent(inout) :: ncFileInfo
-!#! integer,                intent(in)    :: my_ncid
-!#!
-!#! ncFileInfo%ncid = my_ncid
-!#!
-!#! end subroutine nc_set_ncid
-!#!
-!#!
-!#! !------------------------------------------------------------------
-!#! !> get netcdf file id
-!#!
-!#!
-!#! function nc_get_ncid(ncFileInfo)
-!#!
-!#! type(netcdf_file_type), intent(in)    :: ncFileInfo
-!#! integer :: nc_get_ncid
-!#!
-!#! nc_get_ncid = ncFileInfo%ncid
-!#!
-!#! end function nc_get_ncid
-
-
 !------------------------------------------------------------------
 !> return whether the init routines have been called?
 !> or whether it's been marked as using a single file?
 !>@todo fixme: document what this routine does.
 
+
```

Comment:
The rest of the file is formatted with two blank lines before a routine declaration.